src/components/Avatar/Avatar.tsx

```diff
@@ -3,9 +3,18 @@ import { AvatarProps } from './types';
 import React from 'react';
 import guy from './images/guy';
 
-const Avatar: React.FC<AvatarProps> = ({ theme, image, text }: AvatarProps) => {
+const Avatar: React.FC<AvatarProps> = ({
+    theme,
+    image,
+    text,
+    rounded = false,
+}: AvatarProps) => {
     return (
-        <AvatarStyled data-testid="test-avatar">
+        <AvatarStyled
+            data-testid="test-avatar"
+            rounded={rounded}
+            role={`rounded_${rounded}`}
```

Comment:
aria role tag is to provide semantic meaning to an impaired user. Role could be a nice tag to use here, but I would think that `role="Your Avatar"` would provide more meaning to a blind user. What do you think?